pkg/cmd/render/render.go

```diff
@@ -85,28 +115,42 @@ type TemplateData struct {
 // Run contains the logic of the render command.
 func (r *renderOpts) Run() error {
 	renderConfig := TemplateData{}
+
+	// XXX: What is this file used for?
 	if len(r.clusterConfigFile) > 0 {
 		_, err := ioutil.ReadFile(r.clusterConfigFile)
 		if err != nil {
 			return err
 		}
 		// TODO I'm thinking we parse this into a map and reference it that way
 	}
+
 	if err := r.manifest.ApplyTo(&renderConfig.ManifestConfig); err != nil {
 		return err
 	}
 	if err := r.generic.ApplyTo(
 		&renderConfig.FileConfig,
-
-		// TODO I don't think we have these
 		genericrenderoptions.Template{},
 		genericrenderoptions.Template{},
-
 		&renderConfig,
-		nil,
+		nil, // XXX: Can this be used to merge resources and write out config file? (map[string]resourcemerge.MergeFunc)
```

Comment:
it can. But afaik we don't do that anywhere, but keep the config semantics plain and simple.